app/actuators/ResultCard.js

```diff
@@ -627,8 +626,8 @@ export default class ResultCard extends Component {
 		}
 
 		return (
-			<div className="rbc rbc-resultcard" ref={(node) => { this.listContainer = node; }}>
-				<div ref={(div) => { this.listParentElement = div; }} className={`rbc-resultcard-container card thumbnail ${cx}`} style={this.props.componentStyle} style={this.getComponentStyle()}>
```

Comment:
There was a little glitch about `this.props.componentStyle` being used here and also inside `getComponentStyle()` method.

Reply:
You are merging `style` and `this.props.style` here, that shouldn't happen. You should keep both separate and add `this.props.style` at the root level